client/src/Components/Search/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { SearchOutlined } from '@ant-design/icons';
+import Button from '../Button';
+import Select from '../Select';
+
+import './style.css';
+
+import { locations, serviceTypes } from '../../Utils/data';
+
+const Filter = ({ handleSearch }) => {
+  const [service, setService] = useState('null');
+  const [location, setLocation] = useState('null');
+
+  handleSearch(service, location);
+
+  const handleService = (value) => {
+    setService(value);
+  };
+  const handleLocation = (value) => {
+    setLocation(value);
+  };
+  return (
+    <div className="filter-container">
+      <Select
+        className="filter-select"
+        textDefault={serviceTypes[0]}
+        handleChange={handleService}
+        options={serviceTypes}
+      />
+      <p className="split-text">in</p>
+      <Select
+        className="filter-select"
```

Comment:
you can set the className as default on a select component

Reply:
I don't think I understand what you mean but these classes are for a already set style.